### ISSUE TITLE
Pod and Static install conflict with FMDB version

### ISSUE THREAD

**Issue by ghost** [mapbox/DEPRECATED-mapbox-ios-sdk#561]
I have a current project that needs offline map functionality.  Went to install the map box-ios-sdk via Pod and it could not satisfy the FMDB requirement:

[!] Unable to satisfy the following requirements:
- `FMDB (= 2.4)` required by `Podfile.lock`
- `FMDB (~> 2.4.0)` required by `AWSiOSSDKv2 (2.0.13)`
- `FMDB (~> 2.4.0)` required by `AWSiOSSDKv2/SQS (2.0.13)`
- `FMDB (~> 2.4.0)` required by `AWSiOSSDKv2/SES (2.0.13)`
- `FMDB (~> 2.4.0)` required by `AWSiOSSDKv2/Kinesis (2.0.13)`
- `FMDB (~> 2.4.0)` required by `AWSiOSSDKv2/SimpleDB (2.0.13)`
- `FMDB (~> 2.4.0)` required by `AWSiOSSDKv2/SNS (2.0.13)`
- `FMDB (~> 2.4.0)` required by `AWSiOSSDKv2/AWSCore (2.0.13)`
- `FMDB (~> 2.4.0)` required by `AWSiOSSDKv2/EC2 (2.0.13)`
- `FMDB (~> 2.4.0)` required by `AWSiOSSDKv2/CloudWatch (2.0.13)`
- `FMDB (~> 2.4.0)` required by `AWSiOSSDKv2/AutoScaling (2.0.13)`
- `FMDB (~> 2.4.0)` required by `AWSiOSSDKv2/S3 (2.0.13)`
- `FMDB (~> 2.4.0)` required by `AWSiOSSDKv2/DynamoDB (2.0.13)`
- `FMDB (~> 2.4.0)` required by `AWSiOSSDKv2/ElasticLoadBalancing (2.0.13)`
- `FMDB (= 2.3)` required by `Mapbox-iOS-SDK (1.2.0)`

Seems like AWSiOSSDKv2 wants FMDB 2.4.0 and map box wants FMDB 2.3.  So I tried the Static install, and when I compile my project I get 40+ duplicate symbols - the map box lib duping symbols with the existing pod lib (all dupes are from the FMDB package of each lib).  Seems that Objective-C and namespaces is a no go eh?

Any work arounds for 3rd party libraries butting heads like this?

**Comment by aidanlister** [comment]
Yeah two pretty major libraries that don't like each other :( Is there a workaround?

**Comment by jrichau** [comment]
I am seeing the same conflict - anyone figure out a way around?

**Comment by joeytat** [comment]
I'm having the same problem. Anyone solved this ?

**Comment by davidm49090** [comment]
This podspec is still creating conflicts with Pods using FMDB 2.5 including SpatialDBKit. Could we bump up the FMDB version?
